.travis.yml

```diff
@@ -37,14 +37,17 @@ matrix:
       - PRODUCT_TESTS='-g hdfs_impersonation'
       - CLUSTER_TYPE='singlenode-hdfs-impersonation'
     - env:
-      - PRODUCT_TESTS='-g hdfs_impersonation,authorization'
+      - PRODUCT_TESTS='-g hdfs_impersonation,authorization,storage_formats,cli'
       - CLUSTER_TYPE='singlenode-kerberos-hdfs-impersonation'
     - env:
       - PRODUCT_TESTS='-g ldap_cli'
       - CLUSTER_TYPE='singlenode-ldap'
     - env:
       - MVN_TESTS='-Dtest=TestLocalBinarySpilledQueries,TestPostgreSqlDistributedQueries,TestMySqlDistributedQueries'
       - MVN_MODULES='-pl presto-tests,presto-mysql,presto-postgresql'
+    - env:
+      - PRODUCT_TESTS='-g hdfs_no_impersonation,authorization,storage_formats,cli'
```

Comment:
If you're planning to add `authorization` tests, these should be run only under `singlenode-kerberos-hdfs-impersonation` profile and not under `singlenode-kerberos-hdfs-no-impersonation` profile.